example/src/main/java/io/honeycomb/opentelemetry/android/example/MainActivity.kt

```diff
@@ -73,6 +73,14 @@ private fun onSendSpan(otelRum: OpenTelemetryRum?) {
     span?.end()
 }
 
+private fun onSendMetrics(otelRum: OpenTelemetryRum?) {
+    val otel = otelRum?.openTelemetry
+    val meter = otel?.getMeter("@honeycombio/smoke-test")
+    var counter = meter?.counterBuilder("smoke-test.random.int")?.build()
+
+    counter?.add(Math.random().toLong())
```

Comment:
Is there a reason for using random here? I'd be a little worried about it being flaky in some situations, like 0, or negative numbers, or very large numbers. Can we just put a constant number here instead?

Reply:
Sure, yeah, especially if we add a smoke test for it, we want it to be reliable.